pyomo/core/tests/unit/test_sets.py

```diff
@@ -3003,7 +3003,9 @@ def test_setargs2(self):
         a=Set()
         b=Set(a)
         with self.assertRaisesRegex(
-                TypeError, "Cannot apply a Set operator to an indexed"):
+                #TypeError, "Cannot apply a Set operator to an indexed"):
```

Comment:
Can this commented line be removed?

Reply:
Technically, yes: although I wanted to note that the expected exception from this syntax was changing.  The change in the exception is a (slight) backwards compatibility issue, so wanted to leave a breadcrumb that the exception type had changed.